source/includes/pipelines/_06-scheduling.md.erb

```diff
@@ -87,6 +87,69 @@ json = {
 }
 %>
 
+<a name="update-materials-before-scheduling-flag"></a>
+
+### The `update_materials_before_scheduling` flag
+
+When the `update_materials_before_scheduling` flag is `false`, then GoCD does not run a material update before scheduling the pipeline.
+
+When the `update_materials_before_scheduling` flag is `true` (as it is by default), then a material update is run before the pipeline is scheduled. This can cause new commits to be seen for the materials of the pipeline, causing GoCD to use those commits for scheduling the pipeline.
+
+#### Scenario 1:
+
+Given:
+
+- `update_materials_before_scheduling`: `false`
+- `materials`: empty list
+
+What happens:
+
+- Since the flag is `false`, no material update happens.
```

Comment:
Remove since ..., use only the latter part of the statements? Hopefully, that makes it less content and address your "too verbose" comment?

Reply:
Yes removing the since ..., sounds like a good option. It does not look too verbose to me given it is essential to document these scenarios.